timemachine/cpp/src/kernels/k_rotations.cu

```diff
@@ -134,10 +135,6 @@ void __global__ k_rotate_and_translate_mols(
 
             rotate_coordinates_by_quaternion(local_coords, quat);
```

Comment:
Is this the right behavior? I'm not sure if this is centering the molecule before applying the rotation?

Reply:
Per offline discussion it does seem to be the right behavior, but for clarity should be made to rotate about the origin then add the translation.

Reply:
Fixed up in https://github.com/proteneer/timemachine/pull/1205/commits/c404955a3b250e1499d377598b38db85eb6cf6c5